lib/CreepPlayers/Behaviors/MoveCloser.js

```diff
@@ -0,0 +1,22 @@
+
+var MoveUtils = require('../../Utils/MoveUtils');
+var Base = require('./Base');
+
+/**
+ *
+ * @param creepController @type CreepController
+ * @constructor
+ */
+function MoveCloser(creepController) {
```

Comment:
call this moveCloser dumb, more advanced bots should use a better path system (see https://github.com/bovard/raid/issues/30)